test/remote/gateways/remote_pac_net_raven_test.rb

```diff
@@ -156,15 +156,15 @@ def test_authorize_and_void
   def test_authorize_capture_and_void
     auth = @gateway.authorize(@amount, @credit_card, @options)
     assert capture = @gateway.capture(@amount, auth.authorization)
-    assert void = @gateway.void(capture.authorization, {:pymt_type =>  capture.params['PymtType']})
-    assert_failure void
+    assert void = @gateway.void(capture.authorization, {:pymt_type => capture.params['PymtType']})
+    assert_success void
```

Comment:
That was going to be my guess as well, and whomever wrote the original tests just asserted whatever results they were seeing, but really, voiding results of the capture is perfectly valid, until it's settled overnight that is. For the purposes of this test, there's no reason for void to fail right away.